sklearn/metrics/ranking.py

```diff
@@ -184,6 +185,84 @@ def _binary_average_precision(y_true, y_score, sample_weight=None):
                                  average, sample_weight=sample_weight)
 
 
+def detection_error_tradeoff(y_true, y_score, pos_label=None,
+                             sample_weight=None):
+    """Compute error rates for different probability thresholds
+
+    Note: this implementation is restricted to the binary classification task.
+
+    Parameters
+    ----------
+    y_true : array, shape = [n_samples]
+        True targets of binary classification in range {-1, 1} or {0, 1}.
+
+    y_score : array, shape = [n_samples]
+        Estimated probabilities or decision function.
+
+    pos_label : int, optional (default=None)
+        The label of the positive class
+
+    sample_weight : array-like of shape = [n_samples], optional
+        Sample weights.
+
+    Returns
+    -------
+    fps : array, shape = [n_thresholds]
+        A count of false positives, at index i being the number of negative
+        samples assigned a score >= thresholds[i]. The total number of
+        negative samples is equal to fps[-1] (thus true negatives are given by
+        fps[-1] - fps).
+
+    fns : array, shape = [n_thresholds]
+        A count of false negatives, at index i being the number of positive
+        samples assigned a score < thresholds[i]. The total number of
+        positive samples is equal to tps[-1] (thus false negatives are given by
+        tps[-1] - tps).
+
+    thresholds : array, shape = [n_thresholds]
+        Decreasing score values.
+
+    References
+    ----------
+    .. [1] `Wikipedia entry for Detection error tradeoff
+            <https://en.wikipedia.org/wiki/Detection_error_tradeoff>`_
+    .. [2] `The DET Curve in Assessment of Detection Task Performance
+            <http://www.itl.nist.gov/iad/mig/publications/storage_paper/det.pdf>`_
```

Comment:
This link doesn't work. It appears to be a problem on `https://www.nist.gov/` site though.